app/models/merchant.rb

```diff
@@ -8,4 +8,11 @@ class Merchant < ApplicationRecord
   enum status: {"Enabled" => 0, "Disabled" => 1}
 
   
+  def top_5_customers
+    #binding.pry
+    self.customers.select("customers.*, count(*)").joins(:transactions)
```

Comment:
I think this is close but not quite there.  We need to make sure we're only pulling in successful transactions, which isn't happening here. I also think if you adjust the select statement to create a new row (potentially called transaction_count), then we can call that from within the view below, rather than creating a new method.